application/controllers/handler.js

```diff
@@ -73,6 +73,7 @@ module.exports = {
         }
       })
       .catch((error) => {
+        console.log('Error: ', error);
```

Comment:
In general logging the handled error is enough, because the service logs has the following structure: First lines are manual logs and the last line is the requested route with the response code. With this everyone could match the error to the route and the function in handler.js.
Specific to this function I included more data here as described above.